reference-architecture/gce-cli/ansible/playbooks/roles/openshift-ansible-installer/tasks/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: download openshift ansible installer from github
+  git:
+    repo: https://github.com/openshift/openshift-ansible.git
+    dest: '{{ playbook_dir }}/../../../../../openshift-ansible'
```

Comment:
I'm not sure that we should be doing this outside of origin installs. If a version/branch is updated before RPMs are cut we may get into an issue with support. Although our installations are not supported by RH support but the resulting OpenShift environment is required to be supported by RH. I think by using git we may be setting ourselves up for some future issues